lib/jobs/add-misc-node-property-job.js

```diff
@@ -0,0 +1,44 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+module.exports = AddMiscNodePropertyFactroy;
+
+var di = require('di');
+
+di.annotate(AddMiscNodePropertyFactroy, new di.Provide('Job.Add.Misc.Node.Properties'));
```

Comment:
I think it's probably better named like `Job.AddNodeMiscProperties`, the periods usually denote some sort of hierarchical classification.